packages/fxa-settings/src/pages/Signin/index.test.tsx

```diff
@@ -39,174 +39,174 @@ jest.mock('../../lib/glean', () => ({
-
-    expect(signinHeader).toBeInTheDocument();
-    expect(pocketLogo).toBeInTheDocument();
-    expect(passwordInputForm).not.toBeInTheDocument();
-
-    // Pocket links should always open in a new window (announced by screen readers)
-    const pocketTermsLink = screen.getByRole('link', {
-      name: 'Terms of Service Opens in new window',
-    });
-    const pocketPrivacyLink = screen.getByRole('link', {
-      name: 'Privacy Notice Opens in new window',
-    });
-
-    expect(pocketTermsLink).toHaveAttribute(
-      'href',
-      'https://getpocket.com/tos/'
-    );
-    expect(pocketPrivacyLink).toHaveAttribute(
-      'href',
-      'https://getpocket.com/privacy/'
-    );
-  });
-
-  it('emits the expected metrics on render', () => {
-    renderWithLocalizationProvider(
-      <Signin
-        email={MOCK_EMAIL}
-        isPasswordNeeded={false}
-        serviceName={MOCK_SERVICE}
-      />
-    );
-    expect(usePageViewEvent).toHaveBeenCalledWith(viewName, REACT_ENTRYPOINT);
-  });
-
-  it('emits an event on forgot password link click', async () => {
-    renderWithLocalizationProvider(
-      <Signin
-        email={MOCK_EMAIL}
-        isPasswordNeeded={false}
-        serviceName={MOCK_SERVICE}
-      />
-    );
-
-    fireEvent.click(screen.getByText('Forgot password?'));
-
-    await waitFor(() => {
-      expect(GleanMetrics.login.forgotPassword).toBeCalledTimes(1);
-    });
-  });
-
-  describe('form submission', () => {
-    describe('with a password', () => {
-      it('emits the expected metrics events', async () => {
-        renderWithLocalizationProvider(
-          <Signin
-            email={MOCK_EMAIL}
-            isPasswordNeeded={true}
-            serviceName={MOCK_SERVICE}
-          />
-        );
-        submit();
-        await waitFor(() => {
-          expect(GleanMetrics.login.submit).toHaveBeenCalledTimes(1);
-          // the submission handling code hasn't been implemented when the
-          // event was added so a success will follow a a submit.
-          // TODO
-          expect(GleanMetrics.login.success).toHaveBeenCalledTimes(1);
-        });
-      });
-    });
-
-    describe('without a password', () => {
-      it('emits the expected metrics events', async () => {
-        renderWithLocalizationProvider(
-          <Signin
-            email={MOCK_EMAIL}
-            isPasswordNeeded={false}
-            serviceName={MOCK_SERVICE}
-          />
-        );
-        submit();
-        await waitFor(() => {
-          expect(GleanMetrics.cachedLogin.submit).toHaveBeenCalledTimes(1);
-          // the submission handling code hasn't been implemented when the
-          // event was added so a success will follow a a submit.
-          // TODO
-          expect(GleanMetrics.cachedLogin.success).toHaveBeenCalledTimes(1);
-          expect(GleanMetrics.login.submit).toHaveBeenCalledTimes(0);
-        });
-      });
-    });
-  });
-});
+// describe('Signin', () => {
```

Comment:
Same question, is this supposed to be commented out?

Reply:
No, I'm working on the tests and am about to push a stories update. I did this because the stack stopped building for me when I restarted and it got annoying having to fix the types for a build every time when props etc. were in-progress.